app/src/main/java/comp3350/mealbuddy/objects/UserInfo.java

```diff
@@ -18,7 +18,7 @@
     public ActivityLevel activityLevel;
     public Sex sex;
 
-    public User(String fullname, String username, String password, double weight, double height, ActivityLevel activityLevel, Sex sex) {
+    public UserInfo(String fullname, String username, String password, double weight, double height, ActivityLevel activityLevel, Sex sex) {
```

Comment:
We probably want an age

Reply:
Or enums like 5-10, 10-12, 12-14, 14-16, 16-20, 20-25, 25-30 etc

Reply:
It depends on how we do the calculation, for simplicity, I say we get this merged and when someone comes to the point of calculating stuff. It'll be easy to factor that in. Since we don't know if it'll be easier to do the calculations with an exact age or age groups and what those age groups should be, etc.